GOVERNANCE.md

```diff
@@ -364,12 +366,10 @@ Astro features are discussed using a model called [Consensus-seeking decision-ma
 **Who can vote:** All [TSC](#technical-steering-committee-tsc) members.
 
 1. Anyone can submit an RFC to suggest changes to Astro.
-2. A trivial change can be discussed and approved entirely within the RFC GitHub issue, as long as there are no objections from Core or TSC members. This is not considered a formal vote.
-3. A non-trivial, significant change should be discussed within the RFC and approved during an RFC meeting call. In some cases, an RFC may be approved outside of an RFC meeting using Pull Request reviews as a proxy for votes.
-4. During an RFC meeting, the person leading the call will attempt to achieve consensus on the RFC proposal.
-5. **If consensus is reached:** the RFC is approved.
-6. **If consensus is not reached:** The RFC author and TSC members must make all reasonable attempts to resolve issues and reach consensus in GitHub or a follow-up RFC meeting. The process of reaching consensus can take time, and should not be rushed as long as all participants are making a reasonable effort to respond.
-7. **If consensus still cannot be reached:** The project Steward may invoke [rough consensus](https://en.wikipedia.org/wiki/Rough_consensus) to resolve an RFC that has not achieved absolute consensus, as described below (borrowed from the [IETF](https://datatracker.ietf.org/doc/html/rfc2418)):
+2. Changes can be discussed and approved entirely within the RFC GitHub issue. TSC members can vote on the RFC via a review of either **Approve (For)** or **Change Requested (Against)**, in the case of Pull Requests. For issues, TSC members can vote via a comment.
```

Comment:
> TSC members can vote on the RFC via...

Do you mean "must" vote? Would we count votes outside of the PR, for example in a Discord channel?

Reply:
Yes, I meant must vote via PR / issues, will update.

Reply:
I think the wording here should reflect that the vote must be in the issue or PR, but without implying that it's not allowed to discuss it elsewhere!

Reply:
> Yes, I meant must vote via PR / issues, will update.

It doesn't look like this was updated?

> without implying that it's not allowed to discuss it elsewhere!

I do still think we should emphasize that discussion should take place in the PR, though, for the greatest visibility and accessibility. Large parts of our community are not on Discord, and to not reflect important discussion on the PR is leaving them out.

Reply:
I've changed it to `must`.